From 33fd3ca1e2a4c6431cfefc4c8ec6fe40c4df057f Mon Sep 17 00:00:00 2001
From: PixelyIon <pixelyion@protonmail.com>
Date: Sat, 20 Nov 2021 12:55:51 +0530
Subject: [PATCH 02/14] Allow building using `add_subdirectory` with base CMake

yuzu defines a CMake function `create_target_directory_groups` which we don't intend to utilize and are removing from here. This shouldn't affect anything.
---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 545d69c..ba0d3c1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -256,5 +256,3 @@ else()
         $<$<CXX_COMPILER_ID:Clang>:-fbracket-depth=1024>
     )
 endif()
-
-create_target_directory_groups(shader_recompiler)
-- 
2.38.1

